cucumber/authenticators_jwt/features/authn_jwt.feature

```diff
@@ -130,3 +130,58 @@ Feature: JWT Authenticator - JWKs Basic sanity
     """
     CONJ00100E Annotation, 'custom-claim', is empty
     """
+
+  Scenario: Ignore invalid annotations
+    Given I have a "variable" resource called "test-variable"
+    Given I extend the policy with:
+    """
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw: invalid
+        authn-jwt: invalid
+        authn-jwt/raw/sub/sub: invalid
+        authn-jwt/raw2/sub: invalid
+        authn-jwt/raw/sub: valid
+        authn-jwt/raw/namespace-id: valid
```

Comment:
@Avi-Eyal  just for the sake of testing.. try to have it combined.. i.e. - 
valid
not valid
valid
not valid
valid